lib/fabrik/DirectorManager.js

```diff
@@ -211,6 +211,31 @@ class DirectorManager extends BaseManager {
                 previous_manifest: manifest
               }, opts.context);
               return;
+            })
+            .then(() => {
+              let updateActionContext = _.cloneDeep(actionContext);
+              _.chain(updateActionContext)
+                .set('instance_guid', this.getInstanceGuid(deploymentName))
+                .value();
+              _.chain(updateActionContext.params)
+                .set('service_id', this.service.id)
```

Comment:
can you pass previous plan id as well ? Since the services might want to know if the update is from old plan to new plan ? Or can they check from their existing deployment's manifest ?

Reply:
params object, which is part of cf request, has previous_values (service_id, plan_id, org_id, space_id). previous plan can be extracted as `updateActionContext .params.previous_values.plan_id`

Reply:
ok , thanks for the clarification .